cmd/fyne/internal/mobile/build_androidapp.go

```diff
@@ -76,6 +76,10 @@ func goAndroidBuild(pkg *packages.Package, bundleID string, androidArchs []strin
 		if err := mkdir(filepath.Dir(libAbsPath)); err != nil {
 			return nil, err
 		}
+		// If building release and no ldflags are set then remove the useless debug and DWARF build options
+		if release && buildLdflags == "" {
+			buildLdflags = "-s -w"
```

Comment:
I am getting more confused by the current change now. How does this change affect the `goBuild` at line 83? It seems this change is not affecting anything.

Moreover, I am still skeptical about whether we should heuristically assume that building a release does not need debug and DWARF info or not. From the gomobile's original command design, ldflags could be customized as bypass to the underlying go command, whereas fyne in this case, wraps the entire gomobile command and does not have this ability.

Instead of preassuming a `-s -w` for release, we might better to figure out a way to allow users to customize this themselves, such as to allow "-s -w" in `fyne build` (similarly, `-x` etc.) What do you think? @andydotxyz

Reply:
well, there are few reason: <br>
1. in ordinary OS(windows, linux, mac) it look like
![ordinary OS](https://user-images.githubusercontent.com/61238240/167785960-5bd49282-7fc3-4cc4-b5f2-6339e9e3a58b.png)
2. I do not want to be conflict with 
![IOS build](https://user-images.githubusercontent.com/61238240/167786405-0aafc35b-a36f-4d9e-868d-830627136930.png)
3. it has effect on 
![LDFLAGS](https://user-images.githubusercontent.com/61238240/167786840-a6d3fa16-d399-4d20-9f74-984e92547780.png)

Reply:
Ah. I see. Thanks for the clarification!

Reply:
I think that customising ldflags is not something we need to be exporting until there is a compelling reason to.
Anything that ties us to implementation details through public API makes me nervous.
In general releases are intended to ship to stores so we keep them lean.
We may need to consider the option of generating DSYM or equivalent to suppose debugging a stripped binary in a future version.